### ISSUE TITLE
Newest Update: Themes

### ISSUE THREAD

**Issue by Aetherinox** [lukasbach/obsidian-code-files#21]
**Describe the bug**
With the recent update, the original theme now looks great on Obsidian Dark Themes. However, picking any other theme, makes the text dark red.

**To Reproduce**
- Start sandbox / fresh vault
- Install Code Files plugin
- Restart Obsidian after editing the file extensions list to accept different extensions
- Create new code file (used normal CSS in this example).
- Change theme in plugin settings (also restarted)

**Expected behavior**
Different themes should work.

**Screenshots**

## Default Theme
![R0tEzZa](https://github.com/lukasbach/obsidian-code-files/assets/118329232/ba53b410-0afb-4bd9-b3bc-cb31c99640c1)

### Clouds
![iXaKpjF](https://github.com/lukasbach/obsidian-code-files/assets/118329232/003a33fb-cc0f-46af-912f-776000ac574c)

### Brilliance Black
![u6EJTUi](https://github.com/lukasbach/obsidian-code-files/assets/118329232/3097bdf0-e9dd-441e-81ea-0930c0aafc27)

### Tomorrow Night Eighties
![1IRQiOO](https://github.com/lukasbach/obsidian-code-files/assets/118329232/92911971-7563-4ec8-bec2-833e1e2f2ff8)

Also tried restarting Obsidian after a theme change.

**Additional context**
- Windows 11 Pro
- Obsidian Sandbox v1.5.3
- Code Files v1.1.7
- No other plugins installed, fresh sandbox, default theme.

**Comment by lukasbach** [comment]
Thanks for the notice, there was something wrong with how themes with transparent backgrounds were created. It should be fixed now, you don't need to update the plugin.

Note that Cloud is a bright VS Theme, so it will still look weird unless you change to a bright theme or uncheck the "Overwrite background with Obsidian background" in the plugin settings, just as a heads up.

**Comment by Aetherinox** [comment]
Yeah, it appears after I turned off the transparent background setting, I can see the changes now.
Question, does the theme update the background, or does it also apply a new style to the code for variables, functions, classes, etc.

Because for dark themes, I appear to get the same color for text, except the background is different.

![bR3QXFq](https://github.com/lukasbach/obsidian-code-files/assets/118329232/21e2b518-060d-4a07-8a18-c7ac4e0157bc)

![XMSRHTh](https://github.com/lukasbach/obsidian-code-files/assets/118329232/3247fc00-dfe9-4b01-aa75-180d1653e6e9)

The only time I notice an actual change in the text colors is when I go from a dark theme to a light theme.

![YAp5A4F](https://github.com/lukasbach/obsidian-code-files/assets/118329232/efaab783-bafe-4a9b-bfc8-4fb7339a04ec)

**Comment by lukasbach** [comment]
It updates both text token colors and background, though the background changes only apply if the option "Overwrite background with Obsidian background" is unchecked.

You can see how the themes are supposed to look like at https://editor.bitwiser.in/, those are the themes that are used in the plugin. It could very well be that there are some themes that only differ in background color, so it's hard to say if there is an issue in the themes you showed without knowing which exact themes you've tried, but please let me know if there is a theme that you tried that doesn't look like it's supposed to in the theme demo at the bitwiser URL.

**Comment by Aetherinox** [comment]
Ah ok. 

I went through a few themes, and I'd say it appears about 95% accurate. 

Such as with the "LAZY" theme. The colors appear to be very similar, except for the highlight color for the line I've selected. I'll have to test the CSS and see if that's me or some bug, because on mine, a highlighted line is unreadable.

![ybUISHsKaH](https://github.com/lukasbach/obsidian-code-files/assets/118329232/ad028fa7-ea61-42e3-9823-6731c27c6175)

Variables, methods, etc all seem to be changing slightly.
